### PR TITLE
feat: #6478 installations page

### PR DESCRIPTION
**Pull request by willmcvay**
# Pull request checklist

**Detail as per issue below (required):**

fixes: #6478

### CODE REVIEW

File: packages/developer-portal/src/components/pages/analytics-v2/installations/installations-by-app-chart.tsx [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+    const sortedInstallsByApp = installations.data.reduce<{ [key: string]: number }>((current, { appId }) => {
+      const name = apps.data?.find((app) => app && app.id === appId)?.name ?? ''
+
+      if (current[name]) {
+        current[name] += 1
+        return current
+      }
+
+      current[name] = 1
```
Comment (bashleigh) [inline packages/developer-portal/src/components/pages/analytics-v2/installations/installations-by-app-chart.tsx:33]: just avoids extra return in condition

```suggestion
     current[name] = (current[name] || 0) + 1
```

**Review by bashleigh — APPROVED** [review]

File: packages/developer-portal/src/components/pages/analytics-v2/installations/installations-by-app-chart.tsx (hunk 1 above)
Comment (willmcvay) [reply packages/developer-portal/src/components/pages/analytics-v2/installations/installations-by-app-chart.tsx:33]: Agreed on this, will update both reduce functions 👍🏻

File: packages/developer-portal/src/components/pages/analytics-v2/installations/installations-per-day-chart.tsx [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+  }, {})
+
+  const labels = Object.keys(sortedInstallsByDate)
+  const data = labels.map((label) => sortedInstallsByDate[label])
+
+  return {
+    labels,
+    data,
+  }
+}
```
[the remark this answers is not in the document]
Comment (willmcvay) [reply packages/developer-portal/src/components/pages/analytics-v2/installations/installations-per-day-chart.tsx:43]: They both do different things inside the reduce ie one finds an app name, the other formats a date. By the time I've added your above suggestion and broken out the reduce into a generic func, it's probably more code than less!

[the remark this answers is not in the document]
Comment (bashleigh) [reply packages/developer-portal/src/components/pages/analytics-v2/installations/installations-per-day-chart.tsx:43]: fair!

**Pull request merged**